src/js/breadcrumbs/breadcrumbs.underscore

```diff
@@ -0,0 +1,9 @@
+<div class="sr-is-focusable" tabindex="-1"></div>
+<% if (breadcrumbs !== null && breadcrumbs.length > 0) { %>
+    <nav class="breadcrumbs" aria-label="<%- nav_aria_label %>">
+    <% _.each(breadcrumbs, function (breadcrumb) { %>
+        <a class="nav-item" href="<%= breadcrumb.url %>"><%- breadcrumb.title %></a>
```

Comment:
Our best practice is to use ``<%-`` even for URLs (the safe template linter would complain about this).